docs/PROGRESS.md

```diff
@@ -1,25 +1,25 @@
-| Library | Percentage |

```

Comment:
not sure I agree with making them ordered by progress, I think alphabetically is still the way to go here.